testData/preferences/maggie.json

```diff
@@ -0,0 +1,14 @@
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/common/magnification": 2.0,
+                    "http://registry.gpii.net/common/tracking": ["caret"],
```

Comment:
It works for me (honest!).

I think if you set it manually and it still doesn't work, then you can blame Windows.